docs/development/vendoring.md

```diff
@@ -1,24 +1,28 @@
 # Vendoring Go dependencies
 
 kops uses [dep](https://github.com/golang/dep) to manage vendored
-dependencies.
+dependencies in most versions leading up to kops 1.14.
+kops uses [go mod](https://github.com/golang/go/wiki/Modules) to manage
+vendored dependencies in versions 1.15 and newer.
 
 ## Prerequisites
 
 The following software must be installed prior to running the
 update commands:
 
 * [bazel](https://github.com/bazelbuild/bazel)
-* [dep](https://github.com/golang/dep)
+* [dep](https://github.com/golang/dep) or go >= v1.12 for go mod support
```

Comment:
can dep still really be used? Gopkg.toml and lock does not exist anymore?

Reply:
You're right, my wording was confusing.  I was trying to say it's one or the other but how about this change.